src/main/java/frc/robot/drive/DriveCommands.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.drive;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.util.AllianceUtil;
+
+public class DriveCommands {
```

Comment:
I wanted to keep that out of the subsystem because I wanted the subsystem to have zero year-specific implementation. I think the best solution other than this would be a turn command producer in the subsystem and pulling all of the angles out to constants if we don't want to go for the DriveCommands class.

The other consideration is should there be a turn to angle command that gets triggered by a button if we move to the heading drive idea?

Reply:
I would argue that no commands to an evergreen subsystem like the drive should have year-specific implementation. Instead it should be paramaterized to inputs.